index.html

```diff
@@ -4,6 +4,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="shortcut icon" type="image/png" href="./images/favicon/">
```

Comment:
Better add a favicon link after the title. I think all links should be in the same place

Reply:
good notice. Done 👍